lib/parsers/webhook_json_parser.rb

```diff
@@ -87,8 +87,6 @@ def deleted?
           @data['push']['changes'][0]['closed']
         when 'tfs'
           @data['resource']['refUpdates'][0]['newObjectId'] == '0000000000000000000000000000000000000000'
-        else
-          false
```

Comment:
Why would you remove this? I actually like being able to rely on a boolean.

Reply:
It was unreachable. We already raise an exception if we can't detect a VCS.  I can put it back though.